src/components/Learn.js

```diff
@@ -11,10 +11,12 @@ import { LinearGradient } from 'expo-linear-gradient'
 const { width, height } = Dimensions.get('window')
 const SHOP_IMAGE_RATIO = 599/416
 const SHOP_IMAGE_WIDTH = width * .6
-const STORE_OPTION_FONT_SIZE = height * .05
+const STORE_OPTION_FONT_SIZE = height * .03
 const SHOP_IMAGE_HEIGHT = SHOP_IMAGE_WIDTH / SHOP_IMAGE_RATIO
+const ICON_SIZE = width * .08
+const ICON_COLOR = 'white'
 
-const Store = ({ navigation }) => {
+const Learn = ({ navigation }) => {
 	const { t } = useTranslation()
 
   return (
```

Comment:
line 25 we have a coins component, this isn't part of your pr but could be nice to remove it since it doesn't make sense in this part of the app